stf 1408/use community operators (#470)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_importing-dashboards.adoc

. Import the cloud dashboard:
+
[WARNING]

> If you do not set the value of the collectd `virt` plugin parameter `hostname_format` to `name uuid hostname` in the `stf-connectors.yaml` file, some of the panels of the cloud dashboard display no information. For more information about the `virt` plugin, see link:{defaultURL}/operational_measurements/collectd-plugins_assembly[collectd plugins].

Reviewer: This felt awkward to me. Double-negative "do not set"/"no information" perhaps?

Open to other wordings, or even the original one, just a thought.
```suggestion
If you set the collectd `virt` plugin parameter `hostname_format` to a value other than `name uuid hostname` in the `stf-connectors.yaml` file, some of the panels on the cloud dashboard will display no information. For more information about the `virt` plugin, see link:{defaultURL}/operational_measurements/collectd-plugins_assembly[collectd plugins].
```

Reviewer 2: We should avoid saying "will" anyway, because it sounds like it's making a prediction. We should write in the simple present tense as much as possible.

Author: I wonder if we even need this at all any more, because our defaults at https://infrawatch.github.io/documentation/#creating-the-base-configuration-for-stf_assembly-completing-the-stf-configuration call this out.

We might have this here because it's an old section before we added the defaults to the base configuration, or we could just have this here because we kept getting reports when folks weren't setting up data collection quite correctly.

I'd also be fine setting the assertive form though.

> If you do not set the value of the collectd `virt` plugin parameter `hostname_format` to `name uuid hostname` in the `stf-connectors.yaml` file, some of the panels of the cloud dashboard display no information. For more information about the `virt` plugin, see link:{defaultURL}/operational_measurements/collectd-plugins_assembly[collectd plugins].

Author: ```suggestion
In the `stf-connectors.yaml` file, ensure you set the value of the collectd `virt` plugin parameter `hostname_format` to `name uuid hostname`, otherwise some of the panels on the cloud dashboard display no information. For more information about the `virt` plugin, see link:{defaultURL}/operational_measurements/collectd-plugins_assembly[collectd plugins].
```

Author: Using the word from @mickogeary here as it felt the cleanest.

> For some panels in the cloud dashboard, you must set the value of the collectd `virt` plugin parameter `hostname_format` to `name uuid hostname` in the `stf-connectors.yaml` file. If you do not configure this parameter, affected dashboards remain empty. For more information about the `virt` plugin, see link:{defaultURL}/operational_measurements/collectd-plugins_assembly[collectd plugins].

Author: This link was out of date. Fixed to use point at the new home inside of Operational Measurements.

> For some panels in the cloud dashboard, you must set the value of the collectd `virt` plugin parameter `hostname_format` to `name uuid hostname` in the `stf-connectors.yaml` file. If you do not configure this parameter, affected dashboards remain empty. For more information about the `virt` plugin, see link:{defaultURL}/operational_measurements/collectd-plugins_assembly[collectd plugins].

Reviewer 2: ```suggestion
If you do not set the value of the collectd `virt` plugin parameter `hostname_format` to `name uuid hostname` in the `stf-connectors.yaml` file, some of the panels of the cloud dashboard display no information. For more information about the `virt` plugin, see link:{defaultURL}/operational_measurements/collectd-plugins_assembly[collectd plugins].
```
